src/components/NavigationLink/NavigationLink.tsx

```diff
@@ -45,16 +59,29 @@ export default class NavigationLink extends React.Component<NavigationLinkProps>
   }
 
   private getClasses() {
-    const { className, unstyled } = this.props;
+    const { className, unstyled, focus, display } = this.props;
 
     const classes: string[] = ['y-navigationLink'];
     if (unstyled) {
       classes.push('y-navigationLink__unstyled');
     }
+
+    if (focus) {
+      classes.push(`y-focus-${camelCaseToDashed(focus)}`);
```

Comment:
Is it worth doing `camelCaseToDashed`? We're using camelCase in classnames in this repo.

Reply:
It may actually be a BEM modifier, so you could just do `y-focus--${focus}`